### ISSUE TITLE
Gtk-WARNING **: Error loading theme icon

### ISSUE THREAD

**Issue by ghost** [PCSX2/pcsx2#363]
Hi,

These are my specs;
```
PCSX2 1.2.2
Slackware 14.1 x86_64 (multilib)
Openbox 3.5.2
gtk+2-2.24.20
gdk-pixbuf2-2.28.2
wxGTK-2.8.12
```
I compiled pcsx2 1.2.2 multilib on my box and when I first start it I get these messages at the prompt;
```
Interface is initializing.  Entering Pcsx2App::OnInit!
Applying operating system default language...
Command line parsing...
Command line parsed!
Creating UserLocalData folder: /home/foo/.config/pcsx2

(pcsx2:2220): Gtk-WARNING **: Error loading theme icon 'edit-find' for stock: KhÃ´ng thá» táº£i mÃ´-Äun táº£i áº£nh:
/usr/lib64/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so: /usr/lib64/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so: lá»p ELF khÃ´ng ÄÃºng: ELFCLASS64

(pcsx2:2220): Gtk-WARNING **: Error loading theme icon 'text-x-generic' for stock: KhÃ´ng thá» táº£i mÃ´-Äun táº£i áº£nh:
/usr/lib64/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so: /usr/lib64/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so: lá»p ELF khÃ´ng ÄÃºng: ELFCLASS64

(pcsx2:2220): Gtk-WARNING **: Error loading theme icon 'edit-find' for stock: KhÃ´ng thá» táº£i mÃ´-Äun táº£i áº£nh:
/usr/lib64/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so: /usr/lib64/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so: lá»p ELF khÃ´ng ÄÃºng: ELFCLASS64

(pcsx2:2220): Gtk-WARNING **: Error loading theme icon 'text-x-generic' for stock: KhÃ´ng thá» táº£i mÃ´-Äun táº£i áº£nh:
/usr/lib64/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so: /usr/lib64/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so: lá»p ELF khÃ´ng ÄÃºng: ELFCLASS64

(pcsx2:2220): Gtk-WARNING **: Error loading theme icon 'application-exit' for stock: Unable to load image-loading module:
/usr/lib64/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so: /usr/lib64/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so: wrong ELF class: ELFCLASS64
```
If you look at this screen shot below you'll notice two things under the ```Plugins Search Path``` the text is cut off and the button to the right of ```/usr/lib/pcsx2``` is showing up in unusual characters.

http://i.imgur.com/8yo4FC6.jpg

In this screen shot below it's the same thing, the text being cut off and the unusual characters in the button;

http://i.imgur.com/y8ui9hh.jpg

It also mentioned something about gdk-pixbuf and a possible method to fix this by running the command;
```
gdk-pixbuf-query-loaders > /usr/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache
```

I ran the command but when I restarted pcsx2 I still saw the same problems and messages.

I tried several gtk2 themes and different iconsets but it didn't help get rid of these messages.

Is there anything I can do to fix this on my end? When I looked at these icons names;
```
'edit-find' for stock:
'text-x-generic' for stock:
'application-exit' for stock
```
I'm not sure if the word ```stock``` on the end means pcsx2 is looking for these icons in the ```stock``` directory, but all the iconsets I use have these icons placed under ```actions```

So is pcsx2 just looking in the wrong /path? My understanding is ```actions``` is the common path these icons are typically found in...

**Comment by moulei** [comment]
Hi gregory38,

For now I'll just stick with wx2 since there seems to be an issue with wx3 looking in the wrong path...

Unless later you want me to test again...

In the meantime did you remove the icons  from the Menu? There were two icons I use to see before in 1.2.2 under 'System' - 'Exit' and 'Misc' - 'About'. I no longer see icons for ```Exit``` & ```About```

**Comment by moulei** [comment]
Hi gregory38,

I just grabbed the latest git version today, and I'm still seeing the foreign lanugage and the cut off text when first starting pcsx2.

Here's some screen shots below. ```Notice where I place the red dots.```

http://i.imgur.com/MMN1cmO.jpg
http://i.imgur.com/auDyaZz.jpg
http://i.imgur.com/ZXs5bEP.jpg

Let me know if you want me to make a new bug report for this?

**Comment by gregory38** [comment]
```
Let me know if you want me to make a new bug report for this?
```
No. Issue is quite complex, unfortunately I didn't manage yet to reproduce it on my side, so it will take time to fix it.

**Comment by moulei** [comment]
Ok, no new report.

Funny thing is, after running the setup, then as an example I run;

CDVD > Iso Selector > Browse

Then the window comes up, now it's all correct and in English. :)

Hey is that a new CD style? LOL... Trying to minimize words? :) CD/DVD

Keep up the great work! :)

**Comment by turtleli** [comment]
I'll close this issue.

The user (who deleted their account) missed out a "-32" in the command:
`gdk-pixbuf-query-loaders-32 > /usr/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache`
That's why they had 64-bit errors.

All the links are dead so I'm unable to tell whether the other issues are also configuration related issues or actual PCSX2 issues (and if so, whether the issues have been fixed).

I tested it on a Slackware64-current multilib VM and the images load fine, nothing looks out of place. wx3.0 works fine.
